site/docs/how-to-release.md

```diff
@@ -435,13 +435,13 @@ spark-shell \
 
 To verify using Flink, start a Flink SQL Client with the following command:
 ```bash
-wget ${MAVEN_URL}/iceberg-flink-runtime/{{ icebergVersion }}/iceberg-flink-runtime-{{ icebergVersion }}.jar
+wget ${MAVEN_URL}/iceberg-flink-runtime-1.20/{{ icebergVersion }}/iceberg-flink-runtime-1.20-{{ icebergVersion }}.jar
 
 sql-client.sh embedded \
-    -j iceberg-flink-runtime-{{ icebergVersion }}.jar \
-    -j ${FLINK_CONNECTOR_PACKAGE}-${HIVE_VERSION}_${SCALA_VERSION}-${FLINK_VERSION}.jar \
+    -j iceberg-flink-runtime-1.20-{{ icebergVersion }}.jar \
+    -j flink-connector-hive_2.12-1.20.jar \
     shell
-```
+
```

Comment:
oops, I accidentally deleted this line. Submitted https://github.com/apache/iceberg/pull/11487 to fix.